### ISSUE TITLE
Workflow expression tool "calculate numerical value" shows error code and cannot be undone without deleting

### ISSUE THREAD

**Issue by astrovsky01** [galaxyproject/galaxy#19122]
When using the 'calculate numeric parameter value' tool and setting the first param to either user-selected or disable, the workflow bubble turns red and cannot be reset. Tested on both ORG and EU

https://github.com/user-attachments/assets/039cb686-541c-45c5-bb28-f27caedb82dc

**Comment by mvdbeek** [comment]
Yes ... this is all correct ? Set an integer value or make it a workflow parameter, as the message says ?

**Comment by mvdbeek** [comment]
You can, make sure both param values for the two repeats are set.
